packages/manager/modules/support/src/tickets/new-ticket/new-ticket.controller.js

```diff
@@ -86,4 +88,18 @@ export default class SupportNewController {
     this.step = 'issues';
     return this.$q.when();
   }
+
+  isDedicatedServer() {
+    return get(this.serviceType, 'name') === 'dedicated_server';
+  }
+
+  getDedicatedServerErrorTranslationKey() {
```

Comment:
Can we have something more generic ? and more consistent ? Here this function return sometimes a translation key, sometimes a message that doesn't need to be translated, I think we could use `$translate.instant` in it.

Reply:
i return the global message in case where the API status is not yet cover by our side or somethings goes bad.